generate/generate_file_check_test.go

```diff
@@ -121,6 +121,7 @@ func TestCheckStackForGenFile(t *testing.T) {
 			),
 		).String(),
 	)
+	s.ReloadConfig()
```

Comment:
hmm maybe move s.ReloadConfig() inside assertOutdatedFiles ? Seems like we reload for the assert.